data/map reveal.txt

```diff
@@ -1,8 +1,8 @@
 outfit "Galaxy Map"
-	thumbnail "outfit/map"
-	map 1000
-	cost 0
 	category "Special"
+	cost 0
+	map 10000
+	thumbnail "outfit/map"
```

Comment:
It you're trying to match the order of the local map in the vanilla game, the thumbnail should come before the "map" attribute.